### PR TITLE
Beacon Node Docker config

### LINKED ISSUE #300: [Add BN] Docker config
- docker config should have options:
- choose directory where to store chain data (electron native picker)
- eth1 endpoint url
- local port for api
- local port for discovery
- local port for p2p

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #300

### CODE REVIEW

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx [hunk 1]
```diff
@@ -1,30 +1,112 @@
...
+
+    const defaultRpcPort = "5052";
     const [rpcPort, setRpcPort] = useState(defaultRpcPort);
-    const defaultLibp2pPort = ports[0].local;
+
+    const defaultLibp2pPort = "9000";
     const [libp2pPort, setLibp2pPort] = useState(defaultLibp2pPort);
 
+    const defaultDiscoveryPort = "9000";
+    const [discoveryPort, setDiscoveryPort] = useState(defaultLibp2pPort);
```
Comment (mpetrunic) [inline src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:43]: I think we should have default value extracted in some config related to lighthouse

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx [hunk 2]
```diff
@@ -1,30 +1,112 @@
...
+    const [eth1Url, setEth1URL] = useState(defaultEth1URL);
+
+    const defaultRpcPort = "5052";
     const [rpcPort, setRpcPort] = useState(defaultRpcPort);
-    const defaultLibp2pPort = ports[0].local;
+
+    const defaultLibp2pPort = "9000";
     const [libp2pPort, setLibp2pPort] = useState(defaultLibp2pPort);
 
+    const defaultDiscoveryPort = "9000";
```
Comment (mpetrunic) [inline src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:42]: we should add hint somehow that this is udp port

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:43]: another option is to have its own config for all validators and if the user gonna request defaults by validator then we can implement

Comment (mpetrunic) [reply src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:43]: oh but default values are related to beacon node (not validator) like lighthouse, prysm, nimbus, teku etc

Comment (BeroBurny) [reply src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:43]: oh, I mean for beacon nodes :see_no_evil:

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx [hunk 3]
```diff
@@ -1,36 +1,119 @@
...
-import {getNetworkConfig} from "../../services/eth2/networks";
+import {getNetworkConfig, networksList} from "../../services/eth2/networks";
 import {ButtonPrimary} from "../Button/ButtonStandard";
 import {InputForm} from "../Input/InputForm";
+import {Dropdown} from "../Dropdown/Dropdown";
+import {remote} from "electron";
+import {getConfig} from "../../../config/config";
 
-interface IConfigureBNProps {
+export interface IConfigureBNPSubmitOptions {
```
Comment (mpetrunic) [inline src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:11]: what's meaning of "P"

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx (hunk 1 above)
Comment (mpetrunic) [reply src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:43]: I still don't get what you meant :rofl:

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**